source/cloud/nvidia/brev.md

```diff
@@ -49,31 +49,65 @@ Brev Launchables are shareable environment configurations that combine code, con
-   - E.g `https://github.com/NVIDIA-AI-Blueprints/single-cell-analysis-blueprint`
-3. When asked **What type of runtime environment do you need?** select "With container(s)".
+2. When asked **How would you like to provide your code files?**.
+   - Select "I have code files in a git repository", and provide the link to a GitHub repository, if you have one that you'd like
+     to be mounted in the instance once is up.
+   - Otherwise, select "I don't have any code files".
+3. When asked **What type of runtime environment do you need?** select "With container(s)", and proceed.
+
+![Screenshot showing the Brev launchable setup with container](/_static/images/platforms/brev/brev-launchable-setup-start.png)
+
 4. When prompted to **Choose a Container Configuration**, you have two options:
-   1. **If you need a custom container**: Select "Docker Compose" and click on the toggle to select "I have an existing
-      `docker-compose.yaml` file". Under **Upload Docker Compose** select "Provide GitHub/Gitlab URL" and provide a link to
-      a Docker Compose YAML file (e.g. [docker-compose-nb-2506.yaml](https://github.com/NVIDIA-AI-Blueprints/single-cell-analysis-blueprint/blob/main/docker/brev/docker-compose-nb-2506.yaml)).
-      Click "Validate". Note, you needto pass a link to the file in GitHub, not to the `raw.github.com` file.
-   2. **If you need a standard container**: Select "Featured Container" and then select the "NVIDIA RAPIDS" container.
-5. On the next page, when asked **Do you want a Jupyter Notebook experience?**, if you are using a standard container,
-   you will be told there is a preconfigured Jupyter experience. Click "Next". If you are using a custom container with
-   Docker Compose, follow the instructions associated with the Blueprint you are using. Some will include Jupyter pre-installed.
-6. In the section titled **Do you need to expose services?**, if you are using a standard container, you will not need
-   to edit anything. If you are using a custom configuration, make sure to follow the instructions associated with your
-   blueprint.
-7. Select your desired compute environment. Make sure you select sufficient disk size to download the datasets you want
-   to work with. Note, you will not be able to resize the instance once created.
-8. Create a name for your launchable, and deploy 🚀!
+   1. **"Featured Container"** and select the "NVIDIA RAPIDS" container: For a ready to go environment with the
+      entire RAPIDS stack and Jupyter configured.
+      - Select your desired compute environment. Make sure you select sufficient disk size to download the datasets you
+        want to work with. Note, you will not be able to resize the instance once created.
+      - Create a name for your launchable, and deploy.
+   2. **Docker Compose**: For a custom container that you can tailor to your needs.
+      - You can provide a `docker-compose.yaml` via url o from a local file. The structure should be as follow
```

Comment:
I would explicitly instruct people here to "Replace `<name_of_your_github_repo>` with the name of your GitHub repo". Otherwise people might not read it fully.